monitoring/monitorlib/infrastructure.py

```diff
@@ -18,6 +18,7 @@
 
 EPOCH = datetime.datetime.utcfromtimestamp(0)
 TOKEN_REFRESH_MARGIN = datetime.timedelta(seconds=15)
+CLIENT_TIMEOUT = 15  # seconds
```

Comment:
The case where we may need more than 15 seconds is with manual `atproxy` testing -- calls to `atproxy` block indefinitely expecting a worker to query `atproxy` and find the pending request, fulfill it, and post back to `atproxy` with the result.  Doing that manually would usually take more than 15 seconds.  If we could increase this to 60 seconds, I think that use case would be satisfied as well.

Reply:
Thanks for mentioning it. Value changed.